kolibri/core/assets/src/api_resource.js

```diff
@@ -254,7 +411,6 @@ class Resource {
   getModel(id, data = {}) {
     let model;
     if (!this.models[id]) {
-      Object.assign(data, { [this.idKey]: id });
```

Comment:
then I think getModel should not implicitly create a model for me. better return null, and I call createModel explicitly.